README.md

```diff
@@ -164,7 +164,10 @@ All endpoints will require a cryptosuite tag of `ecdsa-rdfc-2019`,
 `ecdsa-jcs-2019`, and/or `ecdsa-sd-2023`. Alongside this cryptosuite tag, you
 will need to specify the `supportedEcdsaKeyTypes` property, parallel to `tags`
 listing the ECDSA key types issuable or verifiable by your implementation.
-Currently, the test suite supports `P-256` and `P-384` ECDSA key types.
+It is recommended to have one issuer per supported Ecdsa KeyType.
+Currently, the `ecdsa-rdfc-2019` test suite supports `P-256` and `P-384` ECDSA key types.
+The `ecdsa-sd-2023` test suite only supports `P-256`.
+Verifier endpoints should support multiple keyTypes and suites.
```

Comment:
The `Ecdsa KeyType `, `ECDSA key types`, and `keyTypes` here should be reconciled. If the lexical differences are intentional, there should be some explanation of what each is intended to convey, as it is unclear to me, and I'm fairly well indoctrinated. It seems likely to significantly confuse an uninitiated reader.

Reply:
@TallTed thank you for the feedback. I have addressed it in the implementations section at least here: https://github.com/w3c/vc-di-ecdsa-test-suite/pull/54/commits/55e66227e0c7641cd36cff66a76765068f475c5c

I went with ECDSA key types